### ISSUE TITLE
(BUG) Paddy works after uninstall - document is infested

### ISSUE THREAD

**Issue by KBUID** [DWilliames/paddy-sketch-plugin#82]
We tried the plugin and its cool so far. Now the following issue is bugging us very very very much.

My colleague used paddy on her already existing symbols. She updated all her symbols after paddy asked her to.

Now **ALL** symbols behave like paddy makes it to - auto spacing after overriding text for example - but many symbols shouldnt rearrange like paddy makes it to. Now here is the real issue.

We uninstalled paddy and the file which got "enhanced" by paddy - still has the feature of paddy!

To be clear, paddy is uninstalled and it still works inside the symbols. This is irreversible! And terrible. Is this a known issue?

Can we fix the document somehow? It seems like paddy code is injected into the document - can we remove the code from the document somehow?

It is very important. Hope there is a fix somehow ...

Regards 

Kevin

EDIT 01

Is this somehow possible as a non nerd? 
https://medium.com/sketch-app-sources/what-does-the-open-sketch-file-format-really-mean-c02a882b6614

My thought is that the plugin itself is bound to the document - because it still runs while being uninstalled(!). We really need to get rid of it in our document :(

**Comment by DWilliames** [comment]
Thanks for reaching out @KBUID. I understand that this must be very frustrating.
There's a couple of things here:

1. **"Is the document injected with Paddy?"** No it isn't. Paddy works as like an overlay to your document – it just resizes elements taking into account how you've named them. But then nothing is embedded into the document itself around Paddy.

2. **"Even after uninstalling the file is still 'enhanced' by Paddy."** This one is extremely weird and concerning. The way Paddy is built, it should not behave like this. After you uninstalled it; your elements shouldn't auto-resize anymore. I have no idea why this is happening – perhaps a bug with Sketch. My only suggestion is restarting the Mac.

3. **"Can we fix the document somehow?"** I know this was 4 days ago now; so I'd be interested to see how you went with it. But the only suggestion I'd have is using 'Undo (CMD + Z).' Or if you have your file on dropbox somewhere you might be able to revert to a previous version. Or even try `File > Revert to > Browse all versions...` and pick a version to go back to.

4. The medium article you linked doesn't appear to be related.

Just to re-iterate; it is not possible for Paddy to 'infest' your document. Uninstalling it should resolve the problem. If it is still persisting; could you please provide a screen recording or something so I can understand the behaviour in more detail.

**Comment by chsWeb** [comment]
Sometimes you have to uninstall the plugin (any plugin) and restart Sketch before everything goes away. Auto Layout, for example, retains the appearance of Stacked Groups even after uninstalling the plugin. The Stacked Groups don't return to normal-looking groups until I restart Sketch.

**Comment by KBUID** [comment]
Thanks for getting back at us. So we discovered after a reboot of sketch - it is now possible to place symbols which have not the usual paddy functionality. The plugin remained deinstalled but the symbols which were applied with paddy functionality still have the paddy behavior. We had to replace all the symbols which was a lot of work but it helped to get it back to normal...

Hope this helps someone in the future - thanks for all the replies.
